src/lib/Microsoft.Health.Events/EventCheckpointing/StorageCheckpointClient.cs

```diff
@@ -92,49 +92,58 @@ public async Task UpdateCheckpointAsync(Checkpoint checkpoint)
             }
         }
 
-        public Task<Checkpoint> GetCheckpointForPartitionAsync(string partitionIdentifier, CancellationToken cancellationToken)
+        public async Task<Checkpoint> GetCheckpointForPartitionAsync(string partitionIdentifier, CancellationToken cancellationToken)
         {
             var prefix = $"{_blobPath}{partitionIdentifier}";
 
-            Task<Checkpoint> GetCheckpointAsync()
+            async Task<Checkpoint> GetCheckpointAsync()
```

Comment:
Curious what benefit we get from having a local delegate for GetCheckpointAsync as opposed to having the contents in line in the method?

Reply:
The delegate was in the previous iteration of the code so I simply kept it. I can simplify things and move the contents inline. @wi-y any objections to that?

Reply:
no objections